desktop/src/apps/ProjectsApp/ProjectsApp.module.css

```
@@ -0,0 +1,530 @@
+  border-radius: 50%;
+  background: var(--color-board-status-closed, #30d158);
+}
+.pvPulse::after {
+  content: "";
+  position: absolute;
+  inset: 0;
+  border-radius: 50%;
+  background: var(--color-board-status-closed, #30d158);
+  animation: pvPulse 2.2s ease-out infinite;
+}
+@keyframes pvPulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "pvPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)